sphinx-doc/source/extensions.rst

```diff
@@ -82,6 +82,7 @@ Examples
 --------
 
 - `CATMAID-synapsesuggestor <https://github.com/clbarnes/CATMAID-synapsesuggestor>`_
+- `CATNAUD-autoproofreader <https://github.com/pattonw/CATMAID-autoproofreader>'_
```

Comment:
CATNAUD? :-)

Reply:
woops, I'll fix it

Reply:
The Collaborative Annotation Tool for Nauseating Amounts of Micrometer-scale Data